### ISSUE TITLE
Fix 'Check ansible syntax' action

### ISSUE THREAD

**Issue by berendt** [osism/testbed#1004]
```
+ ansible-lint -v --force-color
28
Traceback (most recent call last):
29
  File "/usr/local/bin/ansible-lint", line 5, in <module>
30
    from ansiblelint.__main__ import main
31
  File "/usr/local/lib/python3.8/site-packages/ansiblelint/__main__.py", line 37, in <module>
32
    from ansiblelint.generate_docs import rules_as_rich, rules_as_rst
33
  File "/usr/local/lib/python3.8/site-packages/ansiblelint/generate_docs.py", line 6, in <module>
34
    from rich.console import render_group
35
ImportError: cannot import name 'render_group' from 'rich.console' (/usr/local/lib/python3.8/site-packages/rich/console.py)
```

**Comment by berendt** [comment]
https://github.com/ansible-community/ansible-lint/issues/1795

**Comment by berendt** [comment]
Fixed with ec472df55a40ab265cd4301f7c11278d1b047cf1
